chapters/functions.tex

```diff
@@ -1727,6 +1730,9 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 
 {\lstinline!Inline = false, LateInline = true!} is identical to {\lstinline!LateInline = true!}.
 
+The default for late inlining is tool-specific.
+In particular, tools may automatically delay inlining in order to take advantage of function annotations for derivatives and inverses.
+
```

Comment:
I think this would fit even better before the three rules about identical meaning of annotation combinations.

Reply:
now done

Reply:
Looks good.  Resolving.